Makefile

```
@@ -256,7 +255,16 @@ $(SAGE_CONFIGURE): | $(SAGE_ROOT_BUILD)
 
 $(SAGE_ROOT_BUILD): $(cygwin-build)
 	[ -d $(dir $(SAGE_ROOT_BUILD)) ] || mkdir $(dir $(SAGE_ROOT_BUILD))
```

I know this is not part of your PR, but I couldn't help but wonder why this doesn't just do:
```suggestion
	mkdir -p $(dir $(SAGE_ROOT_BUILD))
```